src/libraries/System.Linq.Expressions/ref/System.Linq.Expressions.cs

```diff
@@ -869,26 +870,26 @@ public partial interface IDynamicExpression : System.Linq.Expressions.IArgumentP
     public sealed partial class IndexExpression : System.Linq.Expressions.Expression, System.Linq.Expressions.IArgumentProvider
     {
         internal IndexExpression() { }
+        public int ArgumentCount { get { throw null; } }
         public System.Collections.ObjectModel.ReadOnlyCollection<System.Linq.Expressions.Expression> Arguments { get { throw null; } }
         public System.Reflection.PropertyInfo? Indexer { get { throw null; } }
         public sealed override System.Linq.Expressions.ExpressionType NodeType { get { throw null; } }
         public System.Linq.Expressions.Expression? Object { get { throw null; } }
-        int System.Linq.Expressions.IArgumentProvider.ArgumentCount { get { throw null; } }
         public sealed override System.Type Type { get { throw null; } }
         protected internal override System.Linq.Expressions.Expression Accept(System.Linq.Expressions.ExpressionVisitor visitor) { throw null; }
-        System.Linq.Expressions.Expression System.Linq.Expressions.IArgumentProvider.GetArgument(int index) { throw null; }
+        public System.Linq.Expressions.Expression GetArgument(int index) { throw null; }
```

Comment:
It looks like desktop has this as an explicit implementation.  But looking back at the .NET Core history, it came into .NET Core as an implicit implementation.  Strange.